packages/hash/frontend/src/pages/[shortname]/entities/[entity-uuid].page/create-entity-page.tsx

```diff
@@ -60,7 +60,7 @@ export const CreateEntityPage = ({ entityTypeId }: CreateEntityPageProps) => {
           throw new Error("subgraph not found");
         }
 
-        const draftEntityEditionId: EntityEditionId = {
+        const draftEntityEditionId: EntityVertexId = {
```

Comment:
Can we rename this

Reply:
ba619f9ca2f427265e684138d4d0b9ff0f677a1e